docs/xdump.md

```diff
@@ -292,7 +292,7 @@ The following table shows the events that are available as dump agent triggers:
 | **vmstop**      | The virtual machine stops.                                                  | Exit code; for example, `filter=#129..#192#-42#255`            |
 | **load**        | A class is loaded.                                                          | Class name; for example, `filter=java/lang/String`             |
 | **unload**      | A class us unloaded.                                                        | Not applicable                                                 |
-| **throw**       | An exception is thrown.                                                     | Exception class name; for example, `filter=java/lang/OutOfMem*`|
+| **throw**       | An exception is thrown explicitly in Java code. Use systhrow for unexpected VM exceptions.                                                     | Exception class name; for example, `filter=java/lang/OutOfMem*`|
```

Comment:
@theresa-m - wording is fine but please can you put quotes around systhrow, as has been done for 'throw' in the systhrow section?

Reply:
yes. its updated now